src/algebra/lie/submodule.lean

```diff
@@ -115,6 +115,17 @@ instance : lie_ring_module L N :=
   lie_add     := by { intros x m n, apply set_coe.ext, apply lie_add, },
   leibniz_lie := by { intros x y m, apply set_coe.ext, apply leibniz_lie, }, }
 
+instance module' {S : Type*} [semiring S] [has_scalar S R] [module S M] [is_scalar_tower S R M] :
```

Comment:
If we're doing this for `lie_submodule`s, shouldn't we also do it for `lie_subalgebra`s?

Reply:
Probably, but it seemed I only needed this instance in the course of #10716. Do you want me to add the `lie_subalgebra` instance anyway, or wait until we actually need it?

Reply:
I'd say worth it if you can do it with <= 2 minutes work, otherwise happy to wait.

Reply:
Took a bit longer than that, but added